schemas/context/webinfo.schema.json

```diff
@@ -30,6 +30,14 @@
           "$ref": "https://ns.adobe.com/xdm/context/webreferrer",
           "description":
             "The referrer of a web interaction, which is the URL a visitor came from immediately before the current web interaction was recorded."
+        },
+        "https://ns.adobe.com/xdm/data/metrics/web/link-clicks": {
```

Comment:
I would put this under `xdm:webInteraction`

Reply:
As comment above. are we sure we are ok with having metrics in multiple levels of the hierarchy?

Reply:
Can we move this under `xdm:webInteraction`?

Reply:
done